src/fidesops/ops/api/v1/endpoints/consent_request_endpoints.py

```diff
@@ -133,8 +136,57 @@ def consent_request_verify(
     return _prepare_consent_preferences(db, provided_identity)
 
 
-@router.patch(
+@router.post(
     CONSENT_REQUEST_PREFERENCES,
+    dependencies=[Security(verify_oauth_client, scopes=[CONSENT_READ])],
```

Comment:
Can you add this new scope to the postman collection, the endpoint where create the client?

Also the new scope will need to be exposed in the UI in the clients/ops/admin-ui/src/constants.ts file